### DISCUSSION TITLE
Collection Improvements

### DISCUSSION THREAD

**Discussion by Kofu1221** [night/betterttv#7184] [Ideas]
Collections has been a fantastic addition to bttv; I believe you could make it even better. 

The simple addition of a mass add/remove feature; in essence being able to enable or disable an entire collection with a single click. It'd save so... so........ much time for seasonal changes.

**Comment by night** [comment]
Closing as a duplicate of #7147. Please add your upvote and any extra detail over there so feedback stays consolidated.
